File: C/Lista_01(Estrutura_Sequencial)/Terreno.c
#include <stdio.h>

int main () {

    double larguraTerreno, comprimentoTerreno, valorMetroQuadrado;

    printf("Digite a largura do terreno: ");
    scanf("%lf", &larguraTerreno);

    printf("Digite o comprimento do terreno: ");
    scanf("%lf", &comprimentoTerreno);

    printf("Digite o valor do metro quadrado: ");
    scanf("%lf", &valorMetroQuadrado);

    double areaTerreno = larguraTerreno * comprimentoTerreno;

    double precoTerreno = areaTerreno * valorMetroQuadrado;

    printf("Area do terreno = %.2lf\n", areaTerreno);
    printf("Preco do terreno = %.2lf", precoTerreno);

    return 0;

}

File: Python/Lista_01(Estrutura_Sequencial)/Terreno.py
largura = float(input("Digite a largura do terreno: "))

comprimento = float(input("Digite o comprimento do terreno: "))

valorMetroQuadrado = float(input("Digite o valor do metro quadrado: "))

areaTerreno = comprimento * largura

precoTerreno = areaTerreno * valorMetroQuadrado

print(f"Area do terreno = {areaTerreno:.2f}")
print(f"Preco do terreno = {precoTerreno:.2f}")